### PR TITLE
Introduce `tracing_benches.sh` script

### PR DESCRIPTION
**Pull request by monoid**
It accepts two arguments: `big`/`long` test type and `merge`/`nomerge`
test subtype.  Any of arguments can be replaced with `--all` value.

### CODE REVIEW

File: air/tracing_benches.sh [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        merge|nomerge)
+            subtests=( "$2" )
+            ;;
+        --all)
+            subtests=( merge nomerge )
+            ;;
+        *)
+            error "Unknown subtest type: $2"
+            ;;
+    esac
```
Comment (ValeryAntopol) [inline air/tracing_benches.sh:68]: Incorrect parentheses sequence drives me mad. Maybe it is correct syntax that i don't know, but it is quite hard to understand. Not a request for change, just a notice.

File: air/tracing_benches.sh [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    shift 2
+
+    : "${AIR_WASM_RUNTIME_PATH:=../target/wasm32-wasi/release/air_interpreter_server.wasm}"
```
Comment (ValeryAntopol) [inline air/tracing_benches.sh:71]: Maybe `AIR_INTERPRETER_WASM`? Don't like the `runtime` word here. Also the binary name is `interpreter`, not `runtime`

**Review by ValeryAntopol — APPROVED** [review]
utACK

File: air/tracing_benches.sh (hunk 2 above)
Comment (monoid) [reply air/tracing_benches.sh:71]: This is variable used by the air-trace, and is to be renamed with the related option.

Comment (ValeryAntopol) [reply air/tracing_benches.sh:71]: Yep. Let's change it now in both script and air-trace, or late in a separate PR

File: air/tracing_benches.sh (hunk 1 above)
Comment (monoid) [reply air/tracing_benches.sh:68]: Unfortunately, this is bash's official `case` syntax that has no alternative (except using `if`). Just an informational message :)

**Pull request merged**